README.md

```diff
@@ -127,14 +155,15 @@ INSERT USAGE EXAMPLE HERE
 | Name | Description |
 |------|-------------|
 | <a name="output_nsg_id"></a> [nsg\_id](#output\_nsg\_id) | ID of the network security group attached to the Loader Server nodes |
+| <a name="output_user_data"></a> [user\_data](#output\_user\_data) | n/a |
```

Comment:
This is not in outputs.tf?

Reply:
Same as others - artefact of my debug flow, ty for pointing it out